website/content/docs/k8s/installation/vault/wan-federation.mdx

```diff
@@ -0,0 +1,671 @@
+
+- One time setup in Vault
+  1. Enabling Vault KV Secrets Engine - Version 2.
+  1. Enabling Vault PKI Engine.
+- Setup per Consul datacenter
+  1. Installing the Vault Injector within the Consul datacenter installation
+  1. Configuring a Kubernetes Auth Method in Vault to authenticate and authorize operations from the Consul datacenter
+  1. Enable Vault as the Secrets Backend in the Consul datacenter
+### One time setup on Vault
+1. Enable [Vault KV secrets engine - Version 2](https://www.vaultproject.io/docs/secrets/kv/kv-v2) in order to store the [Gossip Encryption Key](/docs/k8s/helm#v-global-acls-replicationtoken) and the ACL Replication token ([`global.acls.replicationToken`](/docs/k8s/helm#v-global-acls-replicationtoken)).
+
+    ```shell-session
+    $ vault secrets enable -path=consul kv-v2
+    ```
+
+1. Enable Vault PKI Engine in order to leverage Vault for issuing Consul Server TLS certificates.
+
+    ```shell-session
+    $ vault secrets enable pki
+    ```
+
+    ```shell-session
+    $ vault secrets tune -max-lease-ttl=87600h pki
+    ```
+
+### Setup per Consul datacenter
+#### Primary Datacenter (dc1)
+1. Install the Vault Injector in your Consul Kubernetes cluster (dc1), which is used for accessing secrets from Vault. 
+    
+    -> **Note**: In the primary datacenter (dc1), you will not have to configure `injector.externalvaultaddr` value because the Vault server is in the same primary datacenter (dc1) cluster.
+
+    <CodeBlockConfig filename="vault-dc1.yaml" linenumbers lineNumbers highlight="7,8,9">
+
+    ```yaml
+    server:
+      dev:
+        enabled: true
+      service:
+        enabled: true
+        type: LoadBalancer
+    injector:
+      enabled: true
+      authPath: auth/kubernetes-dc1
+    ui:
+      enabled: true
+    ```
+    </CodeBlockConfig>
+
+    ```shell-session
+    $ helm upgrade vault-dc1 --values vault-dc1.yaml hashicorp/vault --wait
+    ```
+
+1. Configure the Kubernetes Auth Method in Vault for the primary datacenter (dc1).
+
+    ```shell-session
+    $ vault auth enable -path=kubernetes-dc1 kubernetes
+    ```
+
+    Because Consul is in the same datacenter cluster as Vault, the Vault Auth Method can use its own CA Cert and JWT to authenticate Consul dc1 service account requests.  Therefore, you do not need to set `token_reviewer` and `kubernetes_ca_cert` on the dc1 Kubernetes Auth Method.
+
+1. Configure Auth Method with Kubernetes API host
+
+    ```shell-session
+    $ vault write auth/kubernetes-dc1/config kubernetes_host=https://kubernetes.default.svc
+    ```
+
+1. Enable Vault as the secrets backend in the primary datacenter (dc1).  However, you will not yet apply the Helm upgrade command.  You will issue the Helm upgrade command after the [Data Integration](/docs/k8s/installation/vault/wan-federation#setup-per-consul-datacenter-1) section.
```

Comment:
@jmurret Should this be the Hem install command versus the Helm upgrade command? I was not sure when I made the suggestion.